src/Directory.Build.props

```diff
@@ -6,9 +6,8 @@
   <Import Project="$(_ParentDirectoryBuildPropsPath)" Condition="Exists('$(_ParentDirectoryBuildPropsPath)')"/>
 
   <PropertyGroup>
-    <NetCoreAppTargetFrameworks>netcoreapp3.1;net5.0</NetCoreAppTargetFrameworks>
-    <StandardTargetFrameworks>netstandard2.0</StandardTargetFrameworks>
-    <MultiTargetFrameworks>$(StandardTargetFrameworks);$(NetCoreAppTargetFrameworks)</MultiTargetFrameworks>
+    <StandardTargetFrameworks>netcoreapp3.1;net5.0</StandardTargetFrameworks>
```

Comment:
Using just `netcoreapp3.1` could be more optimal here - packages that currently target only `netstandard2.0` would then continue being compiled only once.

Reply:
I figured we may want to take advantage of .NET 5 features, but you are right. I wonder if there are any packages which we explicitly wouldn't want to multi-target.